### ISSUE TITLE
Publish to the amazon Appstore

### ISSUE THREAD

**Issue by szaimen** [nextcloud/talk-android#1406]
**Is your feature request related to a problem? Please describe.**
Now that Win11 will get Android app Support, It makes sense to publish the app on the Amazon Appstore since it will improve the discoverability by a lot.

**Describe the solution you'd like**
Publish the App in the Amazon Appstore

cc @AndyScherzinger @mahibi

**Comment by AndyScherzinger** [comment]
Same would apply fro the files app, so also looping in @tobiasKaminsky. I think some years ago we decided to not support it but maybe time sand requirements changed. Not sure if there are any SLAs that might cause work as in constantly burn efforts.

**Comment by tobiasKaminsky** [comment]
I have no idea if/how push notifications will work for normal Android via Amazon Appstore, but at least for Amazon devices we need: https://developer.amazon.com/de/docs/adm/overview.html

Thus I have also no idea how/if this would work on Win11.

**Comment by szaimen** [comment]
Right, what about just reusing (or copying) the f-droid flavor, publish this version to the store and try to implement the different system later?

**Comment by AndyScherzinger** [comment]
> Right, what about just reusing (or copying) the f-droid flavor, publish this version to the store and try to implement the different system later?

Might be an option for the Windows use case while for Amazons Android-based devices I (personally) simply installed the play store.

**Comment by tobiasKaminsky** [comment]
> Right, what about just reusing (or copying) the f-droid flavor, publish this version to the store

Then please also add a big fat warning/info about this :)
Otherwise people will be disappointed.
